### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Screen Reader: Narrator/NVDA is not reading the error message provided for “Work with” field.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3357]
[Check out Accessibility Insight](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)s! - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #NVDA; #Narrator; #Java tooling for IntelliJ/Eclipse; #C+E-H
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
**MAS Violated:** MAS1.3.1
 
**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Give any value in Work with field and remove.
5)Verify whether screen reader is reading the error message or not. 

**Actual Result:** 
Narrator/NVDA is not reading the error message provided for “Work with” field.

 **Expected Result:** 
Narrator/NVDA should read  the error message provided for “Work with” field.
 
**User Impact:** 
Screen Reader may not get the information about the error message if screen reader is not reading it.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
[Bug_3357_Screen Reader Issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451454/Bug_3357_Screen.Reader.Issue.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The install UI is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a screenshot of install AWS Eclipse Toolkits, we could see the same behavior of this issue

![image](https://user-images.githubusercontent.com/12445236/64940834-9bf9a000-d897-11e9-81f1-f2140bcc3c74.png)

**Comment by Flanker32** [comment]
- Importance of experience: Little, users just need to paste our link in the Work with textfield, it's not common to meet issues here and we hadn't received any customer feedback about this issue.
- Quality of experience: Medium, Screen Reader may not get the information about the error message if screen reader is not reading it.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks!

**Comment by Accessibility56** [comment]
External Bug is logged: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551100

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749791
